rustls-platform-verifier/src/verification/apple.rs

```diff
@@ -49,23 +49,19 @@ pub struct Verifier {
     /// Testing only: The root CA certificate to trust.
     #[cfg(any(test, feature = "ffi-testing", feature = "dbg"))]
     test_only_root_ca_override: Option<SecCertificate>,
-    pub(super) crypto_provider: OnceCell<Arc<CryptoProvider>>,
+    pub(super) crypto_provider: Arc<CryptoProvider>,
```

Comment:
I believe this one, and the other fields on different platform providers, no longer need to be `pub(super)`?